SparkPost/2web2ui
# patch/SA-1773 - User applied filters and metrics persist even when a saved report is selected

Patch for [SA-1773](https://sparkpost.atlassian.net/browse/SA-1773)

### What Changed
- Removes baked-in metrics query params to Cypress tests for report builder
- Renames two Cypress tests that (probably) weren't running in CI due to having an invalid file extension
- URL query params merge with report params when updating report builder

### How To Test
- Select a saved report
- Add metrics
- Add filters
- Hard refresh the page (or copy and paste the current URL in to a new tab) and the metrics and filters should persist

### To Do
